patternfly/patternfly-elements
# feat: Updated size of copy icon for Safari display

## Component updates: PFE-Clipboard


Updates relating to a specific component(s).


### Related issues


- #1814 

### What has changed and why


In pfe-clipboard the copy icon displays at the wrong size on Safari. The component template was updated to fix this issue.
- pfe-clipboard.html - updated width and height value for icon--copy svg


### Testing instructions



- [ ] *Test feature 1*
  1. Go here - In Safari /components/clipboard/
  2. Do this - view the examples referring to "copying text"
  3. You should see this - the icon should match the dimensions of the "link" icon in other examples


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [ ] Changelog updated (required for fix and feat changes).
- [ ] Documentation (README.md, WHY.md, etc.) updated or added, if necessary.
- [ ] Link to the demo recording (if applicable).
- [ ] Designer approval.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
